src/main/java/ch/bfh/ti/i4mi/mag/Config.java

```diff
@@ -351,35 +386,36 @@ public OutPayloadLoggerInterceptor soapRequestLogger() {
     public InPayloadLoggerInterceptor soapResponseLogger() {
         return new InPayloadLoggerInterceptor("./logs/[date('yyyyMMdd-HH00')]/[sequenceId]-soap-response.txt");
     }
-    
+
     // ---------------------------------------------
     // Other beans used
     // ---------------------------------------------
-    
+
     @Bean
     public SchemeMapper getSchemeMapper() {
-    	return new SchemeMapper();
+        return new SchemeMapper();
     }
-    
+
     @Bean
     public PatientReferenceCreator getPatientReferenceCreator() {
-    	return new PatientReferenceCreator();
+        return new PatientReferenceCreator();
     }
 
     @Bean
-    public MagCapabilityStatementProvider serverConformanceProvider(
-            final RestfulServer fhirServer,
-            @Value("${mag.baseurl}") final String baseUrl) {
-        return new MagCapabilityStatementProvider(fhirServer, baseUrl);
+    public IServerConformanceProvider<IBaseConformance> serverConformanceProvider(FhirContext fhirContext,
+                                                                                  RestfulServerConfiguration theServerConfiguration) {
+        return new ServerCapabilityStatementProvider(fhirContext, theServerConfiguration);
```

Comment:
Why are you removing the MagCapabilityStatementProvider?

Reply:
Still unsolved.

Reply:
Hi @qligier, 
It seems to be correct in the commit. 
Might be a file diff issue in github?
What can you see if you download the PR delta?

Reply:
Are you sure you have correctly (force-)pushed it? When I clone your branch, I also get the `ServerCapabilityStatementProvider` instead of the `MagCapabilityStatementProvider`.

Reply:
Sorry again, I messed up merging our branch with yours. 
on commit 0020d4 you changed this
`@Bean
    public IServerConformanceProvider<IBaseConformance> serverConformanceProvider(FhirContext fhirContext,
            RestfulServerConfiguration theServerConfiguration) {
        return new ServerCapabilityStatementProvider(fhirContext, theServerConfiguration);
    }`
into this
`@Bean
    public MagCapabilityStatementProvider serverConformanceProvider(
            final RestfulServer fhirServer,
            @Value("${mag.baseurl}") final String baseUrl) {
        return new MagCapabilityStatementProvider(fhirServer, baseUrl);
    }`.

Merging I first amended your change and then restore both.
Fix commit is on the way.